_conferences/android-stream-day-s1-2020.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "Android Stream Day Season 1"
+website: https://www.eventbrite.com/e/android-stream-day-season-1-tickets-112068517954
+location: India
+online: true                # Optional, for online-only events.
+
+date_start: 2020-08-08
+date_end:   2020-08-09
+
+cfp:  # Optional
+  start: 2020-07-02
+  end:   2020-07-20
```

Comment:
The sessionize page states call for speakers ends "15 Jul 2020"

Reply:
Yeah, we've plans to extend that to 20th. Someone from the team will update that within few hours.

Reply:
I think it's now updated in sessionize. @cketti